### ISSUE TITLE
Why is Che Server image the only one where the image name and tag are split into two variables?

### ISSUE THREAD

**Issue by nickboldt** [eclipse-che/che#14188]
Just wondering why for all the images in a custom resource, both the image name/path and its tag can be specified on the same field, but for Che server, the tag is a separate field.

Che server:

```
    # server image used in Che deployment
    cheImage: 'quay.io/nickboldt/airgap-che-server'
    # tag of an image used in Che deployment
    cheImageTag: 'nightly'
```

Everything else:

```
    devfileRegistryImage: 'quay.io/nickboldt/airgap-che-devfile-registry:nightly'
    pluginRegistryImage: 'quay.io/nickboldt/che-pluginregistry:nightly'
    postgresImage: 'quay.io/nickboldt/airgap-postgresql-96-centos7:nightly'
    pvcJobsImage: 'quay.io/nickboldt/airgap-ubi8-minimal:nightly'
    identityProviderImage: 'quay.io/nickboldt/airgap-che-keycloak:nightly'
```

Could the `cheImage` field be improved such that if there's a suffixed `:tag` it's used instead of the value in the `cheImageTag` field? 

If not, could chectl do a preflight check for a tag in the imagename field, and crap out before trying to deploy something like `quay.io/nickboldt/airgap-che-server:nightly:nightly' ?

**Comment by benoitf** [comment]
it seems there are two issues in one

- one for chectl to check the image values before even giving it to che-operator
- one for enhancing che-operator to have a single parameter

**Comment by benoitf** [comment]
maybe operator should check values of the crd and also chectl should have a good way to report these issues.
Fixing it in chectl may be good but let say you're using operatorhub UI or operators you won't have the check so probably needs to be anyway in operator.

**Comment by nickboldt** [comment]
+1 for this being a 7.2 ux improvement fix, not urgent for 7.1

**Comment by tolusha** [comment]
Not sure if it is actual after 
https://github.com/eclipse/che/issues/15874
